docs/rl-algorithms/ddpg.md

```diff
@@ -0,0 +1,195 @@
+1. Clipped surrogate objective (:material-github: [ppo2/model.py#L81-L86](https://github.com/openai/baselines/blob/ea25b9e8b234e6ee1bca43083f8f3cf974143998/baselines/ppo2/model.py#L81-L86))
+1. Value Function Loss Clipping (:material-github: [ppo2/model.py#L68-L75](https://github.com/openai/baselines/blob/ea25b9e8b234e6ee1bca43083f8f3cf974143998/baselines/ppo2/model.py#L68-L75))
+1. Overall Loss and Entropy Bonus (:material-github: [ppo2/model.py#L91](https://github.com/openai/baselines/blob/ea25b9e8b234e6ee1bca43083f8f3cf974143998/baselines/ppo2/model.py#L91))
+1. Global Gradient Clipping (:material-github: [ppo2/model.py#L102-L108](https://github.com/openai/baselines/blob/ea25b9e8b234e6ee1bca43083f8f3cf974143998/baselines/ppo2/model.py#L102-L108)) -->
+
+
+### Experiment results
+
+PR :material-github: [vwxyzjn/cleanrl#137](https://github.com/vwxyzjn/cleanrl/pull/137) tracks our effort to conduct experiments, and the reprodudction instructions can be found at :material-github: [vwxyzjn/cleanrl/benchmark/ddpg](https://github.com/vwxyzjn/cleanrl/tree/master/benchmark/ddpg).
+
+Below are the average episodic returns for [`ddpg_continuous_action.py`](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/ddpg_continuous_action.py) (3 random seeds). To ensure the quality of the implementation, we compared the results against (Fujimoto et al., 2018)[^2].
+
+| Environment      | [`ddpg_continuous_action.py`](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/ddpg_continuous_action.py) | [`OurDDPG.py`](https://github.com/sfujim/TD3/blob/master/OurDDPG.py) (Fujimoto et al., 2018, Table 1)[^2]  | [`DDPG.py`](https://github.com/sfujim/TD3/blob/master/DDPG.py) using settings from (Lillicrap et al., 2016)[^1] in (Fujimoto et al., 2018, Table 1)[^2]    |
+| ----------- | ----------- | ----------- | ----------- |
+| HalfCheetah      | 9260.485 ± 643.088      |8577.29  | 3305.60|
+| Walker2d   | 1728.72 ± 758.33     |  3098.11 | 1843.85 |
+| Hopper   | 1404.44 ± 544.78         |  1860.02 | 2020.46 |
+
+
+
+???+ info
+
+    Note that [`ddpg_continuous_action.py`](https://github.com/vwxyzjn/cleanrl/blob/master/cleanrl/ddpg_continuous_action.py) uses gym MuJoCo v2 environments while [`OurDDPG.py`](https://github.com/sfujim/TD3/blob/master/OurDDPG.py) (Fujimoto et al., 2018)[^2] uses the gym MuJoCo v1 environments. According to the :material-github: [openai/gym#834](https://github.com/openai/gym/pull/834), gym MuJoCo v2 environments should be equivalent to the gym MuJoCo v1 environments.
+
+    Also note the performance of our `ddpg_continuous_action.py` seems to perform worse than the reference implementation on Walker2d and Hopper. This is likely due to :material-github: [openai/gym#938](https://github.com/openai/baselines/issues/938). We would have a hard time reproducing gym MuJoCo v1 environments because they have been long deprecated.
+
+Learning curves:
+
+<div class="grid-container">
+<img src="../ddpg/HalfCheetah-v2.png">
```

Comment:
```suggestion
<img src="ddpg/HalfCheetah-v2.png">
```

Reply:
<img width="1021" alt="image" src="https://user-images.githubusercontent.com/5555347/159189953-e8384bd1-069d-41fe-b86e-476a812e4dd8.png">

I forgot the exact reason but this doesn't work.

Reply:
It works in gitpod.
DId u opened the link under folder ddpg? cuz from the link, ddpg/ddpg/***.png and assuming you are using src ddgp/***.png, then u might be in a wrong folder?

Reply:
Hey maybe I'm missing something obvious, but I tried it with gitpod and it didn't work

<img width="1211" alt="image" src="https://user-images.githubusercontent.com/5555347/159193465-7f843ed9-46a1-41a6-8749-2e3f2176e9b8.png">


How were you previewing your changes? Did you do `mkdocs serve`?

Reply:
I used the preview option at top right corner.

Reply:
I see. The preview option at the top right corner will preview the markdown file, which might differ from the documentation changes.

I have updated the checklist item as 

- [ ]  I have updated the documentation and previewed the changes via `mkdocs serve`.

Reply:
Furthermore, I can improve this docs experience by automatically building the documentation side with GitPod

Reply:
SG

Reply:
SG